### ISSUE TITLE
Bulkload Parts Tool

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/arctos#3214]
In the Bulkload Parts Tool, are we supposed to be able to see the parts loaded by/into other collections? I can see the 193 parts @dustymc loaded into NMMNH:Paleo and it seems like I could delete them or change their status (I haven't tried). 
![image](https://user-images.githubusercontent.com/12507297/99121979-a823df00-25ba-11eb-924d-0cd8768c298b.png)
When I loaded mine (by setting the status to autoload) they just disappeared (i.e. loaded) after a few minutes; is there a reason why these 193 are lingering?

Also, and maybe it's just me, but I feel like when you go to the tool (Enter Data->Batch Tools->Bulkload Parts), it should take you to the 'load .csv' page and not the 'Review and Load' page.

**Comment by dustymc** [comment]
See https://github.com/ArctosDB/arctos/issues/2974 esp https://github.com/ArctosDB/arctos/issues/2974#issuecomment-698601715

**Comment by acdoll** [comment]
Ok. I was mainly making sure that I was supposed to be seeing you entries. 
So, this means that users of new collections (w/ manage_collection) will be able to edit data of their mentors in these component uploaders? And everyone has access to things that you put in this uploader? I guess this is a pretty low risk environment; the worst that could happen is that parts get loaded before they've been reviewed. That's a risk I can live with, but I'm not sure everyone would agree. Perhaps it would be useful to have a GUID_prefix selector to further decrease the chance of any accidental actions?

**Comment by dustymc** [comment]
Short version: yes.

Longer version: it's always been that way, but via the convoluted "claim from folks in your collection" process that never worked very well, and I believe ended up with a bunch of stuff in limbo. This is just transparent - you don't need to claim anything, you can just directly edit stuff that's in your collections, same as anywhere else. I'm just like any other user - you are very free to revoke my access if you don't want me mucking around in your collection (or just don't want to see my messes!).

We could do a guid_prefix selector, but it won't find things loaded against GUID or by UUID. I think that would add confusion and unnecessary complexity, but that's just me. (And some things that use this template - classification loader, for example - don't do anything directly related to catalog records, so it'd be really weird there.)  In any case I intend to get back to sprinkling the "component loader" around in a couple of weeks - please let us know if you disagree with anything I'm saying, just want something different, etc., etc., etc., while this is somewhat contained.

**Comment by acdoll** [comment]
I have used the old 'claimer' for extra attributes and never had much problem with it. I never saw anything in there that wasn't from my volunteers; maybe that was just due to timing. A few things did end up in limbo, but I assumed that was due to me changing things between when they got entered and when I went to upload the extras.
That said, the new system worked just fine for me. I did feel a little uneasy without that validation step, but that's just because I was used to the old way. It does make sense for that to occur automatically and only return the errors. 

> We could do a guid_prefix selector... I think that would add confusion and unnecessary complexity

Agreed (I just don't like complaining about a problem without at least suggesting a solution)
